### PR TITLE
feat: add issue status update

### PR DESCRIPTION
**Pull request by paul-godhouse**

### CODE REVIEW

File: src/kili/presentation/client/issue.py [hunk 1]
```diff
@@ -209,3 +209,16 @@ def issues(
         if as_generator:
             return issues_gen
         return list(issues_gen)
+
+    def update_issue_status(self, issue_id: IssueId, status: IssueStatus) -> Dict[str, Any]:
```
Comment (Jonas1312) [inline src/kili/presentation/client/issue.py:213]:
```suggestion
    def update_issue_status(self, issue_id: str, status: IssueStatus) -> Dict[str, Any]:
```

**Review by Jonas1312 — COMMENTED** [review]
it's not recommended to use NewType custom types for user-facing methods

the SDK user will have type check errors like  this:

```python
from typing import NewType

IssueId = NewType("IssueId", str)


def my_func(issue_id: IssueId) -> None:
    pass


my_func("0123456789")

```

```
Argument of type "Literal['0123456789']" cannot be assigned to parameter "issue_id" of type "IssueId" in function "my_func"
  "Literal['0123456789']" is incompatible with "IssueId"Pylance[reportArgumentType](https://github.com/microsoft/pyright/blob/main/docs/configuration.md#reportArgumentType)
```

File: src/kili/presentation/client/issue.py (hunk 1 above)
Comment (paul-godhouse) [reply src/kili/presentation/client/issue.py:213]: @Jonas1312 I guess it's the same for IssueStatus then?

Comment (Jonas1312) [reply src/kili/presentation/client/issue.py:213]: not it works fine with Literal types

image

You can render the documentation using `mkdocs serve`, and check that your user-facing method accepts native python types.

The user shouldn't have to import anything from `kili.domain.***`

**Pull request merged**